### ISSUE TITLE
[Bug]: CN panic after restart if there is more than one CN replica

### ISSUE THREAD

**Issue by aylei** [matrixorigin/matrixone#5441]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): d22f5bcc3e7c0c162299fbb7c7c203118caa9901
```

### Actual Behavior

CN will panic on restart if there is more than 1 replicas

```
{"level":"INFO","time":"2022/10/08 11:29:34.027168 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists error_info (statement_id varchar(36) not null, span_id varchar(16) not null, node_uuid varchar(36) not null comment \"node uuid in MO, which node accept this request\", node_type varchar(64) not null comment \"node type in MO, enum: DN, CN, LogService;\", err_code varchar(1024) not null, stack varchar(4096) not null, timestamp datetime(26, 6) not null) infile 'etl:system/error_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"DEBUG","time":"2022/10/08 11:29:34.027185 +0800","caller":"trace/schema.go:124","msg":"[Trace] init tables: create cost 3 ms"}
{"level":"INFO","time":"2022/10/08 11:29:34.027228 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 14.878µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.027257 +0800","caller":"compile/scope.go:43","msg":":5 "}
{"level":"INFO","time":"2022/10/08 11:29:34.027490 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 233.046µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.027520 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 82ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.027549 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create database if not exists system_metrics"}
{"level":"INFO","time":"2022/10/08 11:29:34.027665 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 18.61µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.027702 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.028133 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 431.764µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.028164 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 77ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.028193 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.sql_statement_total (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20), type varchar(20)) infile 'etl:system_metrics/sql_statement_total_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.028271 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 19.091µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.028302 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.028726 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 425.49µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.028760 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 91ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.028790 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.sql_transaction_errors (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20), type varchar(20)) infile 'etl:system_metrics/sql_transaction_errors_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.028877 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 22.455µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.028908 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.029357 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 449.479µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.029395 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 81ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.029427 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.sql_statement_errors (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20), type varchar(20)) infile 'etl:system_metrics/sql_statement_errors_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.029507 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 21.157µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.029539 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.030030 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 491.166µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.030066 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 87ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.030098 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.server_connections (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20)) infile 'etl:system_metrics/server_connections_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.030167 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 17.92µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.030214 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.030640 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 426.521µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.030677 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 82ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.030712 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.process_cpu_percent (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20)) infile 'etl:system_metrics/process_cpu_percent_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.030798 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 16.491µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.030829 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.031296 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 467.137µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.031328 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 78ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.031357 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.process_resident_memory_bytes (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20)) infile 'etl:system_metrics/process_resident_memory_bytes_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.031430 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 24.177µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.031460 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.031893 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 434.347µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.031927 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 111ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.031957 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.process_open_fds (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20)) infile 'etl:system_metrics/process_open_fds_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.032060 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 49.129µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.032090 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.032515 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 425.386µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.032549 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 86ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.032580 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.process_max_fds (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20)) infile 'etl:system_metrics/process_max_fds_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.032652 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 16.9µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.032682 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.033116 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 434.8µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.033147 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 86ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.033179 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.sys_cpu_seconds_total (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20)) infile 'etl:system_metrics/sys_cpu_seconds_total_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.033248 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 14.146µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.033278 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.033796 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 518.81µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.033833 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 74ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.033864 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.sys_cpu_combined_percent (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20)) infile 'etl:system_metrics/sys_cpu_combined_percent_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.033942 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 18.433µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.033973 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.034424 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 451.509µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.034458 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 92ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.034488 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.sys_memory_used (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20)) infile 'etl:system_metrics/sys_memory_used_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.034567 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 17.152µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.034597 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.035017 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 419.892µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.035068 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 70ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.035096 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.sys_memory_available (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20)) infile 'etl:system_metrics/sys_memory_available_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.035180 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 20.077µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.035209 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.035616 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 406.918µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.035650 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 78ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.035678 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.sys_disk_read_bytes (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20)) infile 'etl:system_metrics/sys_disk_read_bytes_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.035761 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 24.162µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.035794 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.036248 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 454.614µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.036280 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 80ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.036310 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.sys_disk_write_bytes (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20)) infile 'etl:system_metrics/sys_disk_write_bytes_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.036390 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 24.279µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.036418 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.036826 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 408.446µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.036858 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 67ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.036887 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.sys_net_recv_bytes (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20)) infile 'etl:system_metrics/sys_net_recv_bytes_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"INFO","time":"2022/10/08 11:29:34.036954 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 16.22µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.036983 +0800","caller":"compile/scope.go:43","msg":":6 "}
{"level":"INFO","time":"2022/10/08 11:29:34.037383 +0800","caller":"frontend/mysql_cmd_executor.go:2272","msg":"time of Exec.Run : 401.262µs"}
{"level":"INFO","time":"2022/10/08 11:29:34.037412 +0800","caller":"frontend/mysql_cmd_executor.go:2278","msg":"time of SendResponse 68ns"}
{"level":"INFO","time":"2022/10/08 11:29:34.037439 +0800","caller":"frontend/util.go:459","msg":"Connection id: 74751101 Status: success Statement: create external table if not exists system_metrics.sys_net_sent_bytes (collecttime datetime(26, 6), value double, node varchar(36), role varchar(20), account varchar(20)) infile 'etl:system_metrics/sys_net_sent_bytes_*.csv' fields terminated by , enclosed by \" lines terminated by \n"}
{"level":"DEBUG","time":"2022/10/08 11:29:34.037449 +0800","caller":"metric/metric.go:183","msg":"[Metric] init metrics tables: create cost 9 ms"}
{"level":"INFO","time":"2022/10/08 11:29:34.037696 +0800","caller":"frontend/mysql_cmd_executor.go:798","msg":"User  change database from [] to [mo_catalog]\n"}
{"level":"INFO","time":"2022/10/08 11:29:34.037721 +0800","caller":"frontend/util.go:459","msg":"Connection id: 4294967295 Status: success Statement: use mo_catalog"}
{"level":"INFO","time":"2022/10/08 11:29:34.037741 +0800","caller":"frontend/util.go:459","msg":"Connection id: 4294967295 Status: success Statement: start transaction"}
{"level":"INFO","time":"2022/10/08 11:29:34.038225 +0800","caller":"frontend/session.go:931","msg":"dbName mo_catalog tableNames [mo_database mo_tables mo_columns %!%mo_increment_columns mo_user mo_account mo_role mo_user_grant mo_role_grant mo_role_privs]"}
{"level":"INFO","time":"2022/10/08 11:29:34.039221 +0800","caller":"frontend/session.go:931","msg":"dbName mo_catalog tableNames [mo_database mo_tables mo_columns %!%mo_increment_columns mo_user mo_account mo_role mo_user_grant mo_role_grant mo_role_privs]"}
{"level":"INFO","time":"2022/10/08 11:29:34.039781 +0800","caller":"frontend/mysql_cmd_executor.go:2149","msg":"time of Exec.Build : 2.013954ms"}
{"level":"INFO","time":"2022/10/08 11:29:34.039837 +0800","caller":"compile/scope.go:43","msg":"\t:2 projection(typ:<id:61 width:100 > col:<name:\"mo_database.datname\" > ) -> projection(typ:<id:61 width:100 > col:<name:\"mo_database.datname\" > ) -> pipe connector"}
{"level":"INFO","time":"2022/10/08 11:29:34.039860 +0800","caller":"compile/scope.go:43","msg":"\t:2 projection(typ:<id:61 width:100 > col:<name:\"mo_database.datname\" > ) -> projection(typ:<id:61 width:100 > col:<name:\"mo_database.datname\" > ) -> pipe connector"}
{"level":"INFO","time":"2022/10/08 11:29:34.039872 +0800","caller":"compile/scope.go:43","msg":":0  union all  -> sql output"}
{"level":"INFO","time":"2022/10/08 11:29:34.041617 +0800","caller":"frontend/mysql_cmd_executor.go:2232","msg":"time of Exec.Run : 1.798406ms"}
{"level":"INFO","time":"2022/10/08 11:29:34.041664 +0800","caller":"frontend/util.go:459","msg":"Connection id: 4294967295 Status: success Statement: show databases"}
{"level":"INFO","time":"2022/10/08 11:29:34.041701 +0800","caller":"frontend/authenticate.go:3994","msg":"error: internal error: it must have result set","span":{}}
{"level":"INFO","time":"2022/10/08 11:29:34.041742 +0800","caller":"frontend/util.go:459","msg":"Connection id: 4294967295 Status: success Statement: rollback"}
{"level":"INFO","time":"2022/10/08 11:29:34.041753 +0800","caller":"frontend/mysql_cmd_executor.go:2630","msg":"----close mce"}
panic: internal error: it must have result set

goroutine 90 [running]:
github.com/matrixorigin/matrixone/pkg/cnservice.(*service).createMOServer(0xc0004aa000, {0x63b9a50, 0xc00061e140}, 0xc00078b080)
	/Users/alei/go/src/github.com/matrixorigin/matrixone/pkg/cnservice/server.go:262 +0x1e5
github.com/matrixorigin/matrixone/pkg/cnservice.(*service).initMOServer(0xc0004aa000, {0x63b9a50, 0xc00040e580}, 0xc00078b080)
	/Users/alei/go/src/github.com/matrixorigin/matrixone/pkg/cnservice/server.go:204 +0x170
github.com/matrixorigin/matrixone/pkg/cnservice.NewService(0xc000282840, {0x63b9a50, 0xc00040e580}, {0x63c5d20?, 0xc0004109f0}, {0x63ea420?, 0xc0004005a0}, {0xc0008419e8, 0x1, 0x1})
	/Users/alei/go/src/github.com/matrixorigin/matrixone/pkg/cnservice/server.go:89 +0x40a
main.startCNService.func1({0x63b9a50, 0xc00040e580})
	/Users/alei/go/src/github.com/matrixorigin/matrixone/cmd/mo-service/main.go:136 +0x26e
github.com/matrixorigin/matrixone/pkg/common/stopper.(*Stopper).doRunCancelableTask.func1()
	/Users/alei/go/src/github.com/matrixorigin/matrixone/pkg/common/stopper/stopper.go:257 +0x79
created by github.com/matrixorigin/matrixone/pkg/common/stopper.(*Stopper).doRunCancelableTask
	/Users/alei/go/src/github.com/matrixorigin/matrixone/pkg/common/stopper/stopper.go:252 +0x12c
```

### Expected Behavior

CN should no panic

### Steps to Reproduce

```Markdown
1. Follow https://github.com/matrixorigin/matrixone/blob/main/etc/distributed-mem-engine-example/README.md to start a distributed cluster with 2 CN replicas;
2. Restart an CN
3. Panic observed
```

**Comment by aylei** [comment]
@reusee @zhangxu19830126 PTAL

**Comment by daviszhen** [comment]
The engine at least has internal database like mo_catalog.

**Comment by florashi181** [comment]
test again @sukki37

**Comment by sukki37** [comment]
panic has been resolved（commit-id:92439b1d3f73552ca912de6180099821c39955fd）, but other restart problems still exist.

@aylei you can close this one.
